index.js

```diff
@@ -180,14 +184,14 @@ function format(answers) {
  * @type {Object}
  */
 module.exports = {
-  prompter: function(cz, commit) {
+  prompter: function (cz, commit) {
     cz.prompt.registerPrompt('autocomplete', require('inquirer-autocomplete-prompt'))
     cz.prompt.registerPrompt('maxlength-input', require('inquirer-maxlength-input-prompt'))
 
     getConfig()
       .then(createQuestions)
       .then(cz.prompt)
-      .then(format)
+      .then((answers) => format(answers, configs))
       .then(commit)
-  }
+  },
```

Comment:
I think we can move away from piping functions here.

Assuming we add `async` to the parent function, we can do:
```js
const config = await getConfig()
const questions = createQuestions(config)
const answers = cz.prompt(questions)
const commitMessage = formatCommitMessage(answers, config)

commit(commitMessage)
```